src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplateField/constants.ts

```diff
@@ -0,0 +1,45 @@
+import { ItemResponseType } from 'shared/consts';
+
+export const DisplayModeOptions = {
```

Comment:
Strongly typing this constant will encourage make it easier for future devs to populate this map if they add a new item type in the future: 💪🏻 
```suggestion
export const DisplayModeOptions: Record<
  ItemResponseType,
  { id: string; name: string }[] | undefined
> = {
```